CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ endif()
 #-----------------------------------------------------------------------------
 # Print header
 #-----------------------------------------------------------------------------
-get_repo_version(GC_REPO_VERSION ${CMAKE_CURRENT_SOURCE_DIR})
+# get_repo_version(GC_REPO_VERSION ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Ah, yes. I commented this out because if I don't, I get:
```
CMake Error at src/GCHP_GridComp/GEOSChem_GridComp/geos-chem/CMakeLists.txt:20 (get_repo_version):
  Unknown CMake command "get_repo_version".
``` when I run cmake

Reply:
@LiamBindle has helped me fix this so it's uncommented plus the lines that include the library to avoid the error